packages/nx/src/lock-file/yarn.spec.ts

```diff
@@ -21,7 +21,11 @@ const TypeScriptOnlyPackage = {
 const YargsAndDevkitPackage = {
   name: 'test',
   version: '0.0.0',
-  dependencies: { '@nrwl/devkit': '15.0.13', yargs: '17.6.2' },
+  dependencies: {
+    '@nrwl/devkit': '15.0.13',
+    yargs: '17.6.2',
+    typescript: '4.8.4',
```

Comment:
Typescript is erroneously mapped to ~4.8.2 so we want to explicitly test for this scenario